### PR TITLE
DINT-56: abandonCart event

### PR DESCRIPTION
**Pull request by aniham**
changes:
- update sdk version(need new `abandonCart` event)
- add aep + snowplow handlers for `abandonCart`
- remove references to `eventForwarding.snowplow` (next version of sdk will replace it with `eventForwarding.commerce`)

### CODE REVIEW

File: src/handlers/shoppingCart/abandonCartAEP.ts [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+const XDM_EVENT_TYPE = "commerce.cartAbandons";
+
+const handler = async (event: Event): Promise<void> => {
+    const alloy = await getAlloy();
+    const { shoppingCartContext } = event.eventInfo;
+
+    const payload: BeaconSchema = {
+        eventType: XDM_EVENT_TYPE,
+        commerce: {
+            checkouts: {
```
Comment (jekelly-adobe) [inline src/handlers/shoppingCart/abandonCartAEP.ts:16]: `checkouts` should be `cartAbandons`

File: src/handlers/shoppingCart/abandonCartAEP.ts [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+                id: "1",
+                value: 1,
+            },
+            cart: {
+                cartId: shoppingCartContext.id ?? "1",
```
Comment (jekelly-adobe) [inline src/handlers/shoppingCart/abandonCartAEP.ts:21]: `cartId` should be `cartID`

**Review by jekelly-adobe — CHANGES_REQUESTED** [review]
A few field names in the XDM payload need changing

**Review by jekelly-adobe — APPROVED** [review]

File: src/handlers/shoppingCart/abandonCartAEP.ts [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+
+    const payload: BeaconSchema = {
+        eventType: XDM_EVENT_TYPE,
+        commerce: {
+            cartAbandons: {
+                id: "1",
+                value: 1,
+            },
+            cart: {
+                ID: shoppingCartContext.id,
```
Comment (grahamcrackers) [inline src/handlers/shoppingCart/abandonCartAEP.ts:21]: should it be `id` instead of `ID`? just to keep formatting consistancy?

**Pull request merged**